client/web/web.go

```diff
@@ -318,7 +318,9 @@ func (s *Server) requireTailscaleIP(w http.ResponseWriter, r *http.Request) (han
 		ipv6ServiceHost = "[" + tsaddr.TailscaleServiceIPv6String + "]"
 	)
 	// allow requests on quad-100 (or ipv6 equivalent)
-	if r.Host == ipv4ServiceHost || r.Host == ipv6ServiceHost {
+	if r.Host == ipv4ServiceHost || r.Host == ipv6ServiceHost ||
```

Comment:
Implementation-wise I think it might be more readable to do something like:
```
host := strings.TrimSuffix(r.Host, ":80")
if host == ipv4ServiceHost || host == ipv6ServiceHost {
...
```

Reply:
I agree that just trimming suffix is certainly cleaner.  In general, this approach should be fine.

Reply:
@klyubin I think we're happy to approve and merge this PR after you make this minor change. Please let me know if you don't intend to do that.

Reply:
Sounds good. I made the suggested adjustment. Sorry for the delay.